cypress/e2e/tests/migration/applicationinventory/analysis/python_analysis.test.ts

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+} from "../../../../../utils/utils";
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import { Application } from "../../../../models/migration/applicationinventory/application";
+
+let applicationsList: Array<Analysis> = [];
+describe(["@tier2"], "Python app analysis", () => {
+    before("Login", function () {
+        login();
+        cy.visit("/");
+    });
```

Comment:
we can remove the before block as global beforeEach logs in to before each test.

Reply:
Done, Thanks for noticing, Shveta!